build.py

```diff
@@ -223,6 +223,7 @@ def build_cpp_src(targets: set):
 def run_cargo(cmds):
     ensure_paths()
     env = os.environ.copy()
+    env["PATH"] = f'{env["PATH"]}{os.pathsep}{rust_sysroot}{os.sep}bin'
```

Comment:
Only the following works on my machine:
```suggestion
    env["PATH"] = f'{rust_sysroot}{os.sep}bin{os.pathsep}{env["PATH"]}'
```